opat/comparison.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>OPAT Comparison Tool</title>
+  </head>
+  <main>
+    <frameset cols="*,*,*">
```

Comment:
@mgifford looks like the next missing accessibility item is a `<h1>`

Reply:
Ok, but I'm a bit baffled as to how to do this as the html didn't work when I had a `<body></body>` tag. Have you used framesets before?

Reply:
Let me try it locally.

Reply:
Sorry I meant you have to both `<body>` and `<main>`